### ISSUE TITLE
Dialog box / Popup type window fails to open on Mac

### ISSUE THREAD

**Issue by gogle7** [CollaboraOnline/online#4834]
**Describe the bug**
Dialog box (canvas) doesn't display on Mac (on any browser). Whereas it works fine on Linux and Windows.

**To Reproduce**
1. Open docx document
2. Go to Layout
3. Select watermark 
4. So watermark window doesn't display.
5. After that it just hangs, cannot edit document.

Similarly for excel document as well:
1. Open excel document
2. Select one cell and right click
3. Select Format Cells
4. No dialog window is shown 

**Expected behavior**
Display of respective dialog window. 

![image](https://user-images.githubusercontent.com/40571061/171085062-0728bf5c-eebc-40b2-874f-92c13795e8c8.png)

**Actual behavior**
There is no dialog window. And whenever we select that option, it just hangs and says reconnecting. 

**Desktop**
 - OS:Mac
 - Browser: On any browser (Tested on: Chrome, Firefox, Safari)
 - Version: Latest branch

**Additional context**
It works fine on Linux and windows based OS.

**Comment by gogle7** [comment]
fetched the latest changes, still reproducible with latest commit a17894dbd7c6513d5a26a8f3ad5716b988a1f44a

**Comment by pedropintosilva** [comment]
@thebearon could you test this out please?

**Comment by thebearon** [comment]
Indeed, this is reproducible, but not specific to Mac, but to HiDPI screens instead.
Thanks for reporting!

**Comment by thebearon** [comment]
It's also a regression from the following core commit:
https://cgit.freedesktop.org/libreoffice/core/commit/?id=cf9be3417bc2be5f772c03180b7cbd248b82cad5
```
author		Luboš Luňák <l.lunak@collabora.com>	2022-01-11 19:08:50 +0100
committer	Luboš Luňák <l.lunak@collabora.com>	2022-01-24 14:40:02 +0100

avoid Xlib cairo surfaces for small virtual devices (bsc#1183308)
```

**Comment by vmiklos** [comment]
CC @llunak

**Comment by llunak** [comment]
https://gerrit.libreoffice.org/c/core/+/135588 / https://gerrit.libreoffice.org/c/core/+/135546

**Comment by gogle7** [comment]
Looks like it's fixed by @llunak commits (as above) , can you please confirm and close this issue and also update the compressed build 22.05
So that it can be downloaded from:
```
wget https://github.com/CollaboraOnline/online/releases/download/for-code-assets/core-co-22.05-assets.tar.gz
```
